### PR TITLE
Fix CatalogUI bugged button

### PR DESCRIPTION
**Pull request by parostatkiem**
**Description**

Changes proposed in this pull request:

- #5995 

**Related issue(s)**

### CODE REVIEW

File: resources/service-catalog-addons/charts/service-catalog-ui/values.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 replicaCount: 1
 image:
-  dir:
+  dir: /pr
```
Comment (akucharska) [inline resources/service-catalog-addons/charts/service-catalog-ui/values.yaml]: It seems image is from master. So why adding pr?

**Comment by sjanota** [comment]
Closing in favor of https://github.com/kyma-project/kyma/pull/5950/

**Pull request closed without merging**